tools/uf2conv.py

```diff
@@ -351,8 +372,17 @@ def error(msg):
             try:
                 print("Resetting " + str(args.serial))
                 try:
-                    ser = serial.Serial(args.serial, 1200)
+                    ser = serial.Serial()
+                    ser.port = args.serial
+                    ser.open()
```

Comment:
Minor thing here, but looking at `CheckSerialReset` this initial bps change and `rts` toggle is a no-op.  If the port is already in 1200bps mode, that;s fine. as `_bps === 1200` and the toggle will occur.  I think it's adding 1+ second to every upload without a reason...

Reply:
Assuming 1 transaction per-frame, and the pico is able to handle each frame without any delay,
At most this adds 8ms to the reset, even on USB1.1 low speed. 

low speed/full speed maximum  is 1 packet between 1ms  ± 500ns SOF
125 µs ± 0.0625 µs on high speed.

It ends up closer to 16ms, due to the kernel and rtos scheduling from what I've seen from the USB sniffer.

Reply:
Sure, it's not slow (don't know how long this runs on the Windows USB stack, though, which is always a surprise!).  But I'm saying this change does nothing tothe actual function of the reset.

If the port is already 1200bps and dtr=1, well, the chip's already reset itself before we get here. :)
If the port is already 1200bps and dtr=0, the `ser.dtr=1` will cause reset.
If the port is not at 1200bps, but dtr=1 (for some strange reason), then the bps change will cause a reset.
If the port is not 1200bps and dtr=0, even if the bps change and dtr setting get reordered (ugh, hope not!) it will still work since both the bps and line state callbacks check for reset.

Reply:
The order doesn't matter any more, since as a side-effect of the loop call `tud_task()` as soon as the condition is met can/will hit again, although, this is to satisfy the host side graceful intentional exit.
What PJRC does (correctly) is to watch for a baud rate nobody ever uses anymore --134.5baud.